README.md

```diff
@@ -27,12 +27,31 @@ PEADM is a supported PE module. If you are a PE customer with the standard or pr
 
 ## Overview
 
-This is the standard workflow for installing and using PEADM.
-
-1. To enable the execution of PEADM plans, install Bolt on a jump host with SSH access to all nodes in your installation.
-2. On the Bolt host, run the `peadm::install` plan to bootstrap a new PE installation. For large or extra-large architectures, PEADM creates node groups in the classifier to set relevant parameters in the `puppet_enterprise` module.
-3. Use PE as normal. PEADM is not required until your next upgrade.
-4. When you are ready to upgrade PE, run the `peadm::upgrade` plan from the Bolt host.
+This is the standard workflow for installing PEADM.
+
+1. [Install Bolt](https://www.puppet.com/docs/bolt/latest/bolt_installing) on a jump host.
+2. To create a Bolt project that includes the PEADM module, run:</br>`mkdir <YOUR_PROJECT> && cd <YOUR_PROJECT> && bolt project init <YOUR_PROJECT> --modules puppetlabs-peadm`
```

Comment:
Strictly speaking, the `br` tag should be spelled like this: `<br/>` but this also works.